generators/client/templates/react/webpack/webpack.common.js.ejs

```diff
@@ -140,7 +140,7 @@ module.exports = options => ({
     new ForkTsCheckerWebpackPlugin({ eslint: true }),
     new CopyWebpackPlugin([
       <%_ if (!reactive && (applicationType === 'gateway' || applicationType === 'monolith')) { _%>
-      { from: './node_modules/swagger-ui-dist/*.{js,css,html,png}', to: 'swagger-ui', flatten: true, ignore: ['index.html']},
+      { from: './node_modules/swagger-ui-dist/*.{js,css,html,png}', to: 'swagger-ui', flatten: true, ignore: ['**/index.html']},
```

Comment:
it shouldn't be changed in react as the library version is not upgraded

Reply:
I updated `copy-webpack-plugin` for react and aligned the config so we don't have to fix this next time we upgrade libraries.